_posts/2022-11-02-relicensing.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Relicensing MDAnalysis
+---
+
+<img src="https://www.gnu.org/graphics/lgplv3-with-text-154x68.png"
+title="LGPLv3" alt="LGPLv3 logo" style="float: right"/>
+
+We want to **change the license of MDAnalysis** from the [GNU General
+Public License][GPLv2] (GPL v2+) to the more permissive [Lesser GNU
+Lesser Public License][LGPL] (LGPL v3+) license. Both are *open source
+licenses* but the [LGPL][] gives developers more freedom in how they
+license any of their own code that makes use of MDAnalysis.
+
+The license is the legal basis that determines how code can be copied,
+changed, and incorporated into other code. The license is the only
+reason that one can use someone else's code --- it does not matter if
+the code is publicly available somewhere. If it does not have a
+license you cannot use it legally. Therefore, getting the license
+right is very important.
+
+**In order to change the MDAnalysis license, we need the agreement of
+all our contributors to change from GPL to LGPL.**
+
+In this post we want to share the motivation, outline the relicensing
+process, and invite questions from the community.
+
+
+
+## Why not GPL any more?
+
+Since its initial release in 2008, MDAnalysis has grown from a small
+Python package used by a handful of enthusiastic graduate students and
+postdocs to a mature library that is used by thousands of researchers
+in the molecular sciences. The MDAnalysis library was published under
+an open source license from the start so that anyone could freely use
+it, contribute to it, and build on it. We chose the GNU Public License
+(GPL) for this purpose. Thanks to the "copy-left" portion of the GPL,
+which requires anyone using MDAnalysis in their own code to also adopt
+the GPL for their code, contributors could feel that any time and work
+that they donated and invested into MDAnalysis would not end up
+contributing to software that they could not use themselves in the
+future.
```

Comment:
todo: remove or reword, this is slightly misleading since GPL code can be used in commercial settings

Reply:
Maybe I am misunderstanding your concern. This does not explicitly talk about commercial code. It simply iterates the guarantee of the GPL that code remains free. If the commercial code is under GPL I get to use it under GPL.

Reply:
> would not end up
contributing to software that they could not use themselves

This is the problematic part of the sentence. There is absolutely nothing under GPL that prevents folks from taking your code and using it for their internal purposes. Also code being under GPL doesn't mean it can't be impossible to access, GPL doesn't mean you have a legal obligation to release (so technically it could it behind a SaaS wall or anything else)

Reply:
I understand that there's no requirement to release the code and SaaS is a general problem. But for the cases where you get your hands on code (which is easier for interpreted languages than compiled ones), GPL seems the clearest way to ensure that the code remains accessible. 

I think one problem we have with this post is if it needs to be written for lawyers or for users/developers. Ideally for both... but if I have to come down on one side, then users/developers. You rightly point out that we need to be careful with our language.

Reply:
hopefully if we make it real clear in a disclaimer at the start then it should be fine, but it's one of those things that someone might look at one day and go "but MDAnalysis said this"

I'm currently considering some very bolded wording that says "we aren't lawyers, this is not legal advice, if you want it please contact a lawyer"

Reply:
Loved your disclaimer in PR #251